app/src/components/MessageList.js

```diff
@@ -75,13 +75,13 @@ class MessageList extends Component {
      * significantly different from the current one
      */
     fetchInitialMessages = async () => {
-        const { lastSodOffset } = this.props;
+        const { lastSodOffset, instrument } = this.props;
         const nMessages = MESSAGE_LIST_DEFAULT_PAGE_SIZE * 2 + 1;
         const lowerSodOffset = lastSodOffset - BigInt(MESSAGE_LIST_DEFAULT_PAGE_SIZE - 1);
 
         try {
             const reverseMessagesResponse = await OrderBookService.getMessageList(
-                SNAPSHOT_INSTRUMENT,
```

Comment:
You should remove the matching import, and the constant from the Constants file

Reply:
thanks for pointing out!